pydis_site/templates/events/pages/code-jams/12/rules.html

```diff
@@ -0,0 +1,85 @@
+{% extends "events/base_sidebar.html" %}
+
+{% block title %}Summer Code Jam 2025{% endblock %}
+
+{% block breadcrumb %}
+    <li><a href="{% url "events:index" %}">Events</a></li>
+    <li><a href="{% url "events:page" path="code-jams" %}">Code Jams</a></li>
+    <li><a href="{% url "events:page" path="code-jams/12" %}">Code Jam 2025</a></li>
+    <li class="is-active"><a href="#">Rules</a></li>
+{% endblock %}
+
+{% block event_content %}
+<ol>
+    <li><p>Your solution must use one or more of the pre-approved frameworks. It is not permitted to circumvent this rule by e.g. using the approved framework as a wrapper for another framework.</p></li>
+    <li>
+        <p>
+            <strong>The core of your project must focus on and incorporate "Python in the Browser".</strong>
+        </p>
+    </li>
+    <li><p>Your solution should be platform agnostic. For example, if you use filepaths in your submission, use <code>pathlib</code> to create platform agnostic Path objects instead of hardcoding the paths.</p></li>
+    <li>
+        <p>
+            You must document precisely how to install and run your project.
+            This should be as easy as possible, which means you should consider using dependency managers like <code>pipenv</code> or <code>poetry</code>.
```

Comment:
```suggestion
            This should be as easy as possible, which means you should consider using dependency managers like <code>uv</code> or <code>poetry</code>.
```
With our recent move to `uv` and our existing use of `poetry`, there aren't any situations where I'd recommend pipenv now. For us as reviewers, this is going to make things so much easier when initialising environments.